types/index.d.ts

```diff
@@ -34,23 +34,44 @@ export interface PayPalNamespace {
     version: string;
 }
 
-declare module "@paypal/paypal-js" {
-    export function loadScript(
-        options: PayPalScriptOptions,
-        PromisePonyfill?: PromiseConstructor
-    ): Promise<PayPalNamespace | null>;
+// export type UnknownObject = Record<string, unknown>;
 
-    export function loadCustomScript(options: {
-        url: string;
-        attributes?: Record<string, string>;
-        PromisePonyfill?: PromiseConstructor;
-    }): Promise<void>;
+export function loadScript(
+    options: PayPalScriptOptions,
+    PromisePonyfill?: PromiseConstructor
+): Promise<PayPalNamespace | null>;
 
-    export const version: string;
-}
+export function loadCustomScript(options: {
+    url: string;
+    attributes?: Record<string, string>;
+    PromisePonyfill?: PromiseConstructor;
+}): Promise<void>;
+
+export const version: string;
 
 declare global {
     interface Window {
-        paypal?: PayPalNamespace;
+        paypal?: PayPalNamespace | null;
     }
+    type UnknownObject = Record<string, unknown>;
```

Comment:
Let's avoid exporting `UnknownObject`

Reply:
Should I completely remove that type and use `Record<string, unknown>` only?